website/contributing/release-roles-responsibilites.md

```diff
@@ -55,13 +53,13 @@ Two sub-roles:
 
 ---
 
-## Release Role #2 : Release Copilot
+## Release Role #2 : Community Releasers
 
 ### Details
 
-- 1 or 2 people (the second one being backup)
+- 2 people per each release
```

Comment:
I'd add similar information as we do for Meta releasers, such as:
```
Two sub-roles:
- 1 **release copilot** as main point of contact per minor release (aiming for every 2 months)
- 1 **reverse shadow** per minor release — watches the release closely but mainly serves as backup if release captain is out
```

Reply:
I didn't do it because I think that we wanted both people to be more actively involved and not just one

Reply:
Do we? We at least should define there is a primary point of contact and the secondary copilot should not have to follow the releases as closely as the main one 🤔

Reply:
But even the "2" for Meta includes a mention of the `reverse shadow`. Maybe we could rephrase the `shadow` to emphasise a more active role but there still imho should be a "main copilot" to run the meetings, coordinate with the captain on the releases themselves, etc. The shadow copilot should be involved in most of the discussions, so they always have all the context (and can help with decision-making if necessary) but in general should have different responsibilities than the main copilot imho

Reply:
Yea for Meta's release captains, we could probably get rid of that "reverse shadow" terminology -- it was more about onboarding at that time, but now it feels like we have a more established process. 

Regarding the co-pilot primary/secondary, I think that distinction would be helpful but you two are best to make that call.

Reply:
> unless I misunderstood @cortinico - we want a 2+2 scenario always, not 1+1 (and 1+1 backups)

I think we actually mentioned having the 1+1 and 1+1 backups. It makes for easier onboarding of new folks and gives more responsibility to the single captain/co-pilot.

Reply:
I've updated the Meta section to remove the reverse shadow terminology, and in general leave the two roles as "general descriptions". I've added a longer section at the start mentioning that the crew will identify organically who will drive the effort based on availability.